src/js/components/ArticleEntry.js

```diff
@@ -42,9 +72,13 @@ export default React.createClass({
         </p>
         <div className="ArticleEntry__actions col-md-3 text-right"
              role="group" aria-label="Actions">
-          <Button type="info" size="xs" onClick={this.handleEditClick} icon="pencil" />
-          <Button type="danger" size="xs" onClick={this.handleDeleteClick} icon="trash"
-                  disabled={this.props.status === 2} />
+          <div className="btn-group">
+            <ArchiveButton onClick={this.callAction("archive")} />
+            <MarkAsReadButton onClick={this.callAction("markAsRead")} />
+            <EditButton onClick={this.callAction("edit")} />
+            <DeleteButton onClick={this.callAction("delete", {confirm: true})}
+                          disabled={this.props.status === 2} />
```

Comment:
We probably would like to have a constant here... (this is exactly why @Natim and @ametaireau preferred to have ``deleted=true`` instead of Pocket like ``status=2``, as proposed by @rnewman)